server/game/cards/events/01/wakingthedragon.js

```diff
@@ -0,0 +1,37 @@
+const DrawCard = require('../../../drawcard.js');
+
+class WakingTheDragon extends DrawCard {
+
+    setupCardAbilities() {
+        this.action({
+            title: 'Stand a character',
+            target: {
+                activePromptTitle: 'Select a character',
+                cardCondition: card =>
+                    card.location === 'play area'
+                    && card.getType() === 'character'
+                    && card.isFaction() === 'targaryen'
+                    && card.isUnique()
+                    && card.kneeled
```

Comment:
Waking the Dragon should be able to be played on standing characters (when you just want to return them to hand).